packages/core/data-transfer/lib/providers/local-file-destination-provider.ts

```diff
@@ -88,9 +89,9 @@ class LocalFileDestinationProvider implements IDestinationProvider {
     }
 
     // Encryption
-    // if (options.encryption?.enabled) {
-    //   streams.push(encrypt(options.encryption.key).cipher());
-    // }
+    if (this.options.encryption?.enabled) {
```

Comment:
Do we need to have `?.` here? From what I can tell in the type definition, encryption will always be defined, no?

Reply:
Nope, we can remove it.